tests/test_csc.py

```diff
@@ -0,0 +1,56 @@
+import subprocess
+import asyncio
+
+import asynctest
+
+from lsst.ts import salobj, cbp
+
+STD_TIMEOUT = 15
+
+
+class Harness:
+    def __init__(self, initial_state):
+        salobj.test_utils.set_random_lsst_dds_domain()
+        self.csc = cbp.csc.CBPCSC(initial_state=initial_state)
+        self.remote = salobj.Remote(domain=self.csc.domain, name="CBP", index=0)
+
+    async def __aenter__(self):
+        await self.csc.start_task
+        return self
+
+    async def __aexit__(self, *args):
+        await self.csc.close()
+
+
+class CscTestCase(asynctest.TestCase):
```

Comment:
Consider re-writting this using Salobj's new `BaseCscTestCase`, makes it much cleaner.